conversion.go

```diff
@@ -1,7 +1,82 @@
 // Package conversion TODO(kkweon): Write description
 package conversion
 
+import (
+	"encoding/binary"
+	"errors"
+	"math"
+)
+
+type Endian int
+
+const BigEndian Endian = 0
+const LittleEndian Endian = 1
+
+type Option struct {
+	Endian Endian
+}
+
 // IntToFloat32 converts int to float32
 func IntToFloat32(x int) (float32, error) {
 	return float32(x), nil
 }
+
+// Float32ToBytes converts float32 to []byte with length 4
+func Float32ToBytes(x float32, o *Option) ([]byte, error) {
+	b := make([]byte, 4)
+	ux := math.Float32bits(x)
+	if o == nil || o.Endian == LittleEndian {
```

Comment:
제 생각에는 디폴트는 `BigEndian` 가 좋을 것 같은데요. `LittleEndian` 을 default로 하신 이유가 있나요?

어차피 실제 아키텍쳐랑 상관없기도 한데다 코드 상에서는 BigEndian 을 기본으로 쓰이거든요 (예를 들면 shift 연산들도 아키텍쳐 상관없이 전부 BigEndian 인것처럼 작동)

```suggestion
	if o == nil || o.Endian == BigEndian {
```

Reply:
맞는 말씀이네요. 여기서는 애초에 Endian 이 다른 architecture 간 통신을 고려한 것도 아니니까요.